.github/workflows/release.yaml

```diff
@@ -0,0 +1,514 @@
+          - image: centos:7
+            installCmd: yum -y install epel-release && yum install -y
+            toolRequirements: autoconf automake binutils bzip2 coreutils curl elfutils-devel elfutils-libs findutils gcc gcc-c++ git gmp gmp-devel jq lbzip2 make ncurses ncurses-compat-libs ncurses-devel openssh-clients patch perl pxz python3 sqlite sudo wget which xz zlib-devel patchelf tree
+            DISTRO: CentOS
+            ARTIFACT: "x86_64-linux-centos7"
+          - image: linuxmintd/mint19.3-amd64
+            installCmd: apt-get update && apt-get install -y
+            toolRequirements: libnuma-dev zlib1g-dev libgmp-dev libgmp10 libssl-dev liblzma-dev libbz2-dev git wget lsb-release software-properties-common gnupg2 apt-transport-https gcc autoconf automake build-essential curl ghc gzip libffi-dev libncurses-dev libncurses5 libtinfo5 patchelf tree
+            DISTRO: Mint
+            ARTIFACT: "x86_64-linux-mint19.3"
+          - image: linuxmintd/mint20.2-amd64
+            installCmd: apt-get update && apt-get install -y
+            toolRequirements: libnuma-dev zlib1g-dev libgmp-dev libgmp10 libssl-dev liblzma-dev libbz2-dev git wget lsb-release software-properties-common gnupg2 apt-transport-https gcc autoconf automake build-essential curl ghc gzip libffi-dev libncurses-dev libncurses5 libtinfo5 patchelf tree
+            DISTRO: Mint
+            ARTIFACT: "x86_64-linux-mint20.2"
+    container:
+      image: ${{ matrix.image }}
+    steps:
+      - name: Install requirements
+        shell: sh
+        run: |
+          ${{ matrix.installCmd }} curl bash git ${{ matrix.toolRequirements }}
+
+      - uses: actions/checkout@v3
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: artifacts
+          path: ./out
+
+      - name: Run test
+        run: bash .github/scripts/test.sh
+        env:
+          ARTIFACT: ${{ matrix.ARTIFACT }}
+          DISTRO: ${{ matrix.DISTRO }}
+
+  test-arm:
+    name: Test ARM binary
+    runs-on: ${{ matrix.os }}
+    needs: ["build-arm"]
+    env:
+      TARBALL_EXT: tar.xz
+      DEBIAN_FRONTEND: noninteractive
+      TZ: Asia/Singapore
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - os: [self-hosted, Linux, ARM64]
+            ARTIFACT: "aarch64-linux-ubuntu20"
+            ARCH: ARM64
+    steps:
+      - uses: docker://arm64v8/ubuntu:focal
+        name: Cleanup (aarch64 linux)
+        with:
+          args: rm -rf .ghcup/ cabal/ dist-newstyle/ out/ store/
+
+      - name: git config
+        run: |
+          git config --global --get-all safe.directory | grep '^\*$' || git config --global --add safe.directory "*"
+        shell: bash
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: artifacts
+          path: ./out
+
+      - uses: docker://hasufell/arm64v8-ubuntu-haskell:focal
+        name: Run test (aarch64 linux)
```

Comment:
Are you the official maintainer here, or is there anything even more officialfor aarch64?

Reply:
I maintain these images:

* https://github.com/haskell/ghcup-hs/blob/master/.github/workflows/docker.yaml
* https://github.com/haskell/ghcup-hs/tree/master/docker